include/freerdp/channels/rdpei.h

```diff
@@ -110,15 +128,17 @@ typedef struct _RDPINPUT_TOUCH_EVENT RDPINPUT_TOUCH_EVENT;
 struct _RDPINPUT_PEN_CONTACT
 {
 	UINT8 deviceId;
-	UINT16 fieldsPresent;
+	UINT16 fieldsPresent; /* Mask of RDPINPUT_PEN_FIELDS_PRESENT values */
 	INT32 x;
 	INT32 y;
-	UINT32 contactFlags;
-	UINT32 penFlags;
-	UINT32 pressure;
-	UINT16 rotation;
-	INT16 tiltX;
-	INT16 tiltY;
+	UINT32 contactFlags; /* See RDPINPUT_CONTACT_FLAG*  */
+	UINT32 penFlags;     /* Present if RDPINPUT_PEN_CONTACT_PENFLAGS_PRESENT, values see
+	                        RDPINPUT_PEN_FLAGS */
+	UINT32 orientation;  /* Present if RDPINPUT_PEN_CONTACT_ROTATION_PRESENT, In degree, [0-359] */
+	UINT32
```

Comment:
would be nice to split the comment on 2 lines, so that `pressure` and `UINT32` end on the same line